Makefile

```diff
@@ -5,14 +5,23 @@ NAME := efes
 build:
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o $(NAME)
 
+build-docker:
+	docker build -t efes-base -f ./Docker/efes-base/Dockerfile .
+	$(eval CONTAINER_ID:=$(shell docker create efes-base))
+	mkdir -p ./bin
+	docker cp ${CONTAINER_ID}:/usr/local/bin/efes - > ./bin/efes
+	docker rm -v ${CONTAINER_ID}
+
 up:
-	docker build -t $(NAME) -f Dockerfile .
-	docker-compose rm -fsv
-	docker-compose up --build
+	docker build -t efes-base -f ./Docker/efes-base/Dockerfile .
+	docker-compose -f ./Docker/docker-compose.yml rm -fsv
+	docker-compose -f ./Docker/docker-compose.yml up --build
 
 test:
-	docker-compose rm -fsv
-	docker-compose -f docker-compose-test.yml up --build --exit-code-from test --abort-on-container-exit
```

Comment:
Why did you remove these parameters?

Reply:
I have split this command into two, `docker-compose build` and `docker-compose run test`
Since it runs test container directly, exit-code comes directly from container now.

But I realized that, containers do not stop when the test container stops. So `--abort-on-container-exit` behavior is missing. I will add a command to remove containers afterwards.

Reply:
I switched back to `up` command again since it was more clean.